app/submitter/converter.py

```diff
@@ -76,6 +77,13 @@ def convert_answers(schema, questionnaire_store, routing_path, flushed=False):
         'questionnaire_id': metadata['questionnaire_id'],
     }
 
+    data = {
+        'answers': convert_answers_to_payload_0_0_3(
+            answer_store, list_store, schema, routing_path
+        ),
+        'lists': convert_lists_to_payload_0_0_3(list_store),
```

Comment:
I might be missing something, but doesn't list_store.serialise() give you what you need here? Meaning you can get rid of all the code for convert_lists_to_payload_0_0_3(list_store)
```'lists': list_store.serialise()```

Reply:
Good point - I think there would still need to be some code to filter out `primary_person` key value pairs where the value is None